transifex/requirements.txt

```diff
@@ -1,5 +1,11 @@
-Django==1.11.2
-edx-i18n-tools==0.3.9
+Babel==1.3
+Django==1.10.7
```

Comment:
I talked with @amangano-edx about this a bit.  If there are edx-platform specific requirements it may make sense to refactor them into their own requirements file so we don't downgrade all of the systems versions.   Most IDAs will be ahead of edx-platform.  If they start upgrading to Django 2.0 who knows what could stop working when we decide to run them using 1.10.

For now I think it would be better to pull out edx-platform specific requirements into their own file.